jina/peapods/__init__.py

```diff
@@ -1,22 +1,24 @@
 __copyright__ = "Copyright (c) 2020 Jina AI Limited. All rights reserved."
 __license__ = "Apache-2.0"
 
-from typing import Dict, Union
+from typing import Optional
 
 from .. import __default_host__
 from ..enums import PeaRoleType, RemoteAccessType
 from ..helper import is_valid_local_config_source
 from ..logging import default_logger
+from jina.peapods.peas.gateway import GatewayPea, RESTGatewayPea
 
 if False:
     import argparse
 
 
-def Pea(args: 'argparse.Namespace' = None, allow_remote: bool = True, **kwargs):
-    """Initialize a :class:`BasePea`, :class:`RemoteSSHPea` or :class:`ContainerPea`
+def RuntimePea(args: Optional['argparse.Namespace'] = None,
```

Comment:
Since `runtime` will always have a `Pea`, should we just call this `Runtime`?

Reply:
some runtimes contain pods like remote ones

Reply:
That's why I make Pea specific.

Reply:
I think `Runtime` sounds better as we have `BaseRuntime` in the previous comments